tests/unit/test.program.js

```diff
@@ -323,4 +324,13 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
+  it('returns the development package version', () => {
+    process.env.NODE_ENV = 'development';
+    let root = path.join(__dirname, '..', '..');
+    let git = require('git-rev-sync');
+    var devVersion = `${git.branch()}-${git.long()}`;
+    assert.equal(defaultVersionGetter(root),
+                     devVersion);
```

Comment:
`devVersion` should line up visually with the opening of the parenthesis in `assert.equal()` from the line above